src/python/pants/backend/jvm/subsystems/jvm.py

```diff
@@ -5,15 +5,31 @@
 from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
                         unicode_literals, with_statement)
 
-from pants.option.custom_types import list_option
+import logging
+import os
+
+from pants.base.exceptions import TaskError
+from pants.option.custom_types import dict_option, list_option
 from pants.subsystem.subsystem import Subsystem
+from pants.util.memo import memoized_property
 from pants.util.strutil import safe_shlex_split
 
 
+logger = logging.getLogger(__name__)
+
+
 class JVM(Subsystem):
   """A JVM invocation."""
   options_scope = 'jvm'
 
+  _OS_NAMES = {
+    'macos': {'macos', 'darwin', 'macosx', 'mac os x'},
```

Comment:
We have other code for this elsewhere when we bootstrap protoc and other tools.  We should probably consolidate code.